autoscaler/README.md

```diff
@@ -0,0 +1,37 @@
+# Horizontal Self Scaler container
+
+This container image watches over the number of schedulable nodes in the cluster and resizes
+the number of replicas in the required controller.
+
+Usage of pod_nanny:
```

Comment:
I haven't even gotten to code yet and this thing has 3 names.  "horizontal-self-scaler", "autoscaler" and "pod_nanny".  Can we pick one name for the repo and the binary?

pod_nanny is not quite right, because this doesn't interact with pods.  self-scaler isn't right because it's not self any more.

How about "proportional-autoscaler" or "autoscale-by-cluster-size" or "simple-autoscaler" ?